magicblock-task-scheduler/Cargo.toml

```diff
@@ -19,20 +19,14 @@ magicblock-ledger = { workspace = true }
 magicblock-program = { workspace = true }
 magicblock-processor = { workspace = true }
 rusqlite = { workspace = true }
-solana-svm = { workspace = true }
-solana-timings = { workspace = true }
 serde = { workspace = true }
-solana-sdk = { workspace = true }
 solana-program = { workspace = true }
 solana-pubsub-client = { workspace = true }
+solana-rpc-client = { workspace = true }
+solana-rpc-client-api = { workspace = true }
+solana-sdk = { workspace = true }
+solana-svm = { workspace = true }
+solana-timings = { workspace = true }
```

Comment:
But why those dependencies are needed here? Also please use specific solana subcrates instead of sdk, unless you have to import too many of them.